docs/release-notes.md

```diff
@@ -17,7 +17,7 @@ Minor changes:
 - Fix unlikely decompression error reading initrd
 - Add release notes to documentation
 - iso: Detect incomplete ISO files
-- Added Fedora 38 signing key
+- Added Fedora 38 signing key, dropped Fedora 35 key
```

Comment:
In the past, this has been mentioned as a major change.

Let's use the present tense to match the style.

Also, `s/key/signing key/` for consistency?

Reply:
fixed